src/common/k8s-api/kube-api.ts

```diff
@@ -58,14 +58,32 @@ export interface DerivedKubeApiOptions {
   /**
    * If the API uses a different API endpoint (e.g. apiBase) depending on the cluster version,
    * fallback API bases can be listed individually.
+   *
    * The first (existing) API base is used in the requests, if apiBase is not found.
-   * This option only has effect if checkPreferredVersion is true.
+   *
+   * This option only has effect if {@link DerivedKubeApiOptions.checkPreferredVersion} is `true`.
    */
   fallbackApiBases?: string[];
```

Comment:
2 questions:
1. Should we use newly introduced `allowedUsableVersions` for Ingresses as well https://github.com/lensapp/lens/blob/master/src/common/k8s-api/endpoints/ingress-class.api.ts#L16
2. Should we completely remove `fallbackApiBases` entity then?

Reply:
1. Yes
2. No, because that is used for when the `apiBase` changes (not just the version). Take `IngressClassApi` as an example. When it was being tested it was under `/apis/extensions` but now it is under `/apis/networking.k8s.io`. That is what `fallbackApiBases` is for.